pylearn2/termination_criteria/__init__.py

```diff
@@ -221,16 +221,16 @@ def initialize(self, model):
         if self._new_epochs:
             self._epochs_done = 0
```

Comment:
The docstring for this class needs to be updated, because it defines max_epochs in terms of the number of calls to this class's continue_learning method.

Is the `new_epochs` flag actually useful at all? It seems incompatible with push_monitor. How do you do two-stage learning without push_monitor?

Why not infer the number of epochs seen from monitor.get_epochs_seen() rather than maintaining a separate count of epochs inside this class?